test/functional/services/remote/webdriver.ts

```diff
@@ -249,9 +247,7 @@ async function attemptToCreateCommand(
         const session = await new Builder()
           .forBrowser(browserType)
           .setFirefoxOptions(firefoxOptions)
-          .setFirefoxService(
-            new firefox.ServiceBuilder(geckoDriver.path).setStdio(['ignore', input, 'ignore'])
-          )
+          .setFirefoxService(new firefox.ServiceBuilder().setStdio(['ignore', input, 'ignore']))
```

Comment:
Actually a small question: don't we need to pass the geckodriver path here? The docs states the following:

```Path to the server executable to use. If omitted, the builder will attempt to locate the geckodriver on the system PATH.```

We install geckodriver using node_modules will it be able to found if we do not provide the path?

Reply:
I copied the solution from `main`, just backporting those commits - it will work the same way probably as it does on main.